<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="identifier.sqlite" uuid="1926f0ef-7f65-4ed9-aa04-c0e1b265ed61">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:identifier.sqlite</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="315ea4f0-a3de-456f-979f-f81bd764ede4">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/itProger/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\python\Desktop\first\itProger\db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>